source/tomster.html.erb

```diff
@@ -1,20 +1,29 @@
 ---
 title: "Tomster and Zoey"
+filters: all, tomster, zoey
 ---
 
 <div class="tomster section intro">
   <h1>Tomster and Zoey</h1>
   <p>
-    Tomster was the first of many friendly faces of the Ember project and community. Over the years we've dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you'll see Ember friends around even more.
+    Tomster was the first of many friendly faces of the Ember project and community. Over the years we"ve dressed and redressed him for all sorts of fun projects and reasons. With the addition of his sister Zoey, you"ll see Ember friends around even more.
```

Comment:
The quote here changed from a single to a double quote, could you change that back?